modules/order/src/Entity/Order.php

```diff
@@ -422,6 +445,43 @@ public function preSave(EntityStorageInterface $storage) {
   }
 
   /**
+   * {@inheritdoc}
+   */
+  public function collectAdjustments() {
```

Comment:
Ah that's right. I forgot we need a way for the order to get all adjustments from its items. I guess I just got thrown off by the content of the method, thinking we could offload it.